Alamofire.podspec

```diff
@@ -1,12 +1,12 @@
 Pod::Spec.new do |s|
   s.name = 'Alamofire'
-  s.version = '1.1.0'
+  s.version = '1.1.3'
   s.license = 'MIT'
   s.summary = 'Elegant HTTP Networking in Swift'
   s.homepage = 'https://github.com/Alamofire/Alamofire'
   s.social_media_url = 'http://twitter.com/mattt'
   s.authors = { 'Mattt Thompson' => 'm@mattt.me' }
-  s.source = { :git => 'https://github.com/Alamofire/Alamofire.git', :tag => '1.1.0' }
+  s.source = { :git => 'https://github.com/Alamofire/Alamofire.git', :tag => '1.1.3' }
```

Comment:
@kylef could you let me know a link (of a web page or a blog) on specifying the requirement (iOS 8)?

Thanks.